SingularityBase/src/main/java/com/hubspot/singularity/SingularityPendingDeploy.java

```diff
@@ -10,14 +10,16 @@
   private final Optional<SingularityLoadBalancerUpdate> lastLoadBalancerUpdate;
   private final DeployState currentDeployState;
   private final Optional<SingularityDeployProgress> deployProgress;
+  private final Optional<SingularityRequest> newRequestData;
```

Comment:
yeah, went back and forth on the name like 10 times already. Open to changing for sure, `updatedRequest` sounds good